### ISSUE TITLE
Blocks: Alt+F10 should navigate to block toolbar regardless of visibility

### ISSUE THREAD

**Issue by aduth** [WordPress/gutenberg#10907]
Currently, Alt+F10 only works to navigate to the block toolbar if it is visible. When not using the Unified Toolbar setting, the toolbar is made invisible while a user is typing. Thus, the user must take steps to reactivate the block toolbar before being able to press Alt+F10.

**Steps to reproduce:**

1. (Prerequisite) If not already, disable Unified Toolbar
2. Insert paragraph block
3. Press some characters to activate "typing mode", thus making toolbar invisible
4. Press Alt+F10

Expected: Focus is shifted into block toolbar
Actual: Focus is shifted to header toolbar

**Task:**

Alt+F10 should shift focus into the block toolbar, regardless of visibility.

**Technical notes:**

Note that there are multiple toolbars which are meant to respond to the keyboard shortcut. See #10699.

The tricky point is in interoperating with React lifecycle to move focus to a yet-unrendered element. See https://github.com/WordPress/gutenberg/pull/10699#discussion_r226093319 .

**Comment by aduth** [comment]
Related:

- #6302 
- #10699

**Comment by talldan** [comment]
This shortcut was also missing from our docs, so I've made a PR to add it. Not sure if it's worth holding off on merging that till the functionality works as desired.
